internal/subimporter/importer.go

```diff
@@ -619,5 +617,8 @@ func countLines(r io.Reader) (int, error) {
 
 // IsEmail checks whether the given string is a valid e-mail address.
 func IsEmail(email string) bool {
-	return regexEmail.MatchString(email)
+	if _, err := mail.ParseAddress(email); err != nil {
```

Comment:
`mail.ParseAddress()` parses an "address", not an e-mail. For instance `Name <name@email.com>`, `<name@email.com>` and `name@email.com`. However, in this context, we're validating just the e-mail ID component. 

Adding an `email == address.Email` check to the parsed address should address this.

Reply:
Actully we can just check if `email.Name` is not empty to validate if its not the address. For example

```
func IsEmail(email string) bool {
	if em, err := mail.ParseAddress(email); err != nil || em.Name != "" {
              return false
	}
	return true
}
```

Reply:
Doesn't work with `<bécassine@example.com>` though. Angle brackets with no name.

Reply:
Sorry, missed this. It should be `len(addrs)` and not `len(addrs)-1`, right?

Reply:
Yeah, you are right. Doing `email == address.Address` is the better way to do this - https://play.golang.org/p/drvQgi6JdW1

will update PR to that?

Reply:
Yep.

Reply:
Updated the PR.